include/modem/lte_lc.h

```diff
@@ -736,7 +737,10 @@ int lte_lc_psm_req(bool enable);
  * @param active_time Pointer to the variable for parsed active time in seconds.
  *	              Positive integer, or -1 if timer is deactivated.
  *
- * @return Zero on success or (negative) error code otherwise.
+ * @retval 0 if successful.
+ * @retval -EINVAL if input argument was invalid.
+ * @retval -EFAULT if AT command failed.
+ * @retval -EBADMSG if no active time and/or TAU value was received.
```

Comment:
Yeah, it's a challenge that the error codes don't necessarily match. 
Added an extra check in this case so that `-EBADMSG` is returned if active time or TAU is missing

Reply:
We are planning to align those so we can be safe an happy